backend/pkg/handlers/websocket.go

```diff
@@ -119,39 +158,92 @@ func (service *AllDbMethodsWrapper) HandleConnections(w http.ResponseWriter, r *
 				fmt.Println("there is an error with json msg: Websocket")
 				break
 			}
-			//
+			
 			//add chat to database
 			service.repo.AddChatToDatabase(chat)
 			//look for reciever in client list
 			online := false
-			reciever:= make(map[*websocket.Conn]string)
+			reciever := make(map[*websocket.Conn]string)
 			for conn, client := range Clients {
 				if client == chat.Reciever {
 					fmt.Println("chat reciever is online")
-					reciever[conn] = client 
+					reciever[conn] = client
 					online = true
 				}
 			}
 
-			//send Chat message to reciever web conn
+			//send Chat message to channel with reciever web conn
 			if online {
-				service.repo.BroadcastToChannel(BroadcastMessage{WebMessage: WebsocketMessage{Chat: chat, Type:"chat"}, Connections: reciever})
+				service.repo.BroadcastToChannel(BroadcastMessage{
+					WebMessage: WebsocketMessage{
+						Chat: chat, 
+						Type: "chat"}, 
+					Connections: reciever,
+				})
 			} else {
 				//OR
-			//check for chat notif
-			oldChats, count, err := service.repo.CheckForNotification(chat)
-			//add new notif to database or add 1 to count
-			if !oldChats || oldChats && err == nil {
-				service.repo.AddChatNotification(chat, count)
-			}
+				//check for chat notif
+				oldChats, count, err := service.repo.CheckForNotification(chat)
+				//add new notif to database or add 1 to count
+				if !oldChats || oldChats && err == nil {
+					service.repo.AddChatNotification(chat, count)
+				}
 			}
-			
+
 		}
 
 	}
 }
 
 func (r *dbStruct) BroadcastToChannel(msg BroadcastMessage) {
 	fmt.Println("attempting to broadcast")
+	//sends message to channel 
 	r.broadcaster <- msg
+	
+	// message is recieved in handleMessages go routine in server.go 
+}
+
+func (r *dbStruct) ClientsFollowingUser(user *User) map[*websocket.Conn]string {
+
+	list := make(map[*websocket.Conn]string)
+	for conn, name := range Clients {
+		fmt.Println(" check if ", name, "follows ", user.NickName)
+		_, err := r.db.Query(`SELECT COUNT (*) FROM Followers WHERE (followerFName, influencerFName) = (?,?) `, name, user.NickName)
```

Comment:
The Followers table structure has changed to:

CREATE TABLE IF NOT EXISTS "Followers" (
						"follow" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
						"followerID" INTEGER NOT NULL,
						"followerUserName" TEXT NOT NULL,
						"influencerID" INTEGER NOT NULL,
						"influencerUserName" TEXT NOT NULL,
						"accepted" TEXT,
						"unfollow" TEXT,
						FOREIGN KEY(followerUserName)REFERENCES Users(nickName),
						FOREIGN KEY(influencerUserName)REFERENCES Users(nickName)
						);
We can change the db query accordingly.